include/mbgl/map/map.hpp

```diff
@@ -106,8 +108,8 @@ class Map : private util::noncopyable {
     double getScale() const;
     void setZoom(double zoom, std::chrono::steady_clock::duration duration = std::chrono::steady_clock::duration::zero());
     double getZoom() const;
-    void setLonLatZoom(double lon, double lat, double zoom, std::chrono::steady_clock::duration duration = std::chrono::steady_clock::duration::zero());
-    void getLonLatZoom(double &lon, double &lat, double &zoom) const;
+    void setLatLngZoom(LatLng latLng, double zoom, std::chrono::steady_clock::duration duration = std::chrono::steady_clock::duration::zero());
+    void getLatLngZoom(LatLng &latLng, double &zoom) const;
```

Comment:
Yeah, I think so. This was just ported for historical reasons. Good reason to scrap it.